<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="L-BG-P10405304" ignored="0" name="Login Test With Invalid Credentials" tests="1" failures="0" timestamp="03 Aug 2017 16:30:39 GMT" time="0.047" errors="0">
  <testcase name="@BeforeSuite isEnvUp" time="0.694" classname="uk.co.capita.advantagecontractor.basetest.BaseTest">
    <failure type="java.lang.AssertionError" message="AUT with URL -- https://ec2-54-171-188-122.eu-west-123.compute.amazonaws.com/auth/AdvantageLogin.html -- is  DOWN!">
      <![CDATA[java.lang.AssertionError: AUT with URL -- https://ec2-54-171-188-122.eu-west-123.compute.amazonaws.com/auth/AdvantageLogin.html -- is  DOWN!
	at uk.co.capita.advantagecontractor.basetest.BaseTest.isEnvUp(BaseTest.java:69)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeSuite isEnvUp -->
  <testcase name="@AfterTest tearDown" time="0.0" classname="uk.co.capita.advantagecontractor.basetest.BaseTest">
    <skipped/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase name="loginWithInvalidCredTest" time="0.0" classname="uk.co.capita.advantagecontractor.tests.LoginWithInvalidCredTest">
    <skipped/>
  </testcase> <!-- loginWithInvalidCredTest -->
</testsuite> <!-- Login Test With Invalid Credentials -->
